chart/skywalking/values-es6.yaml

```diff
@@ -21,10 +21,10 @@ serviceAccounts:
   oap:
 
 oap:
-  name: skywalking-oap
+  name: oap
```

Comment:
A question, why change skywalking-oap to oap?

Reply:
"skywalking" has been defined in "{{ template "skywalking.name". }}"